File: qlty-cli/tests/lang/php/basic.in/lines.php
<?php
  function f1() {
    $sourceFile = tempnam(sys_get_temp_dir(), "ruby") . ".rb";
    file_put_contents($sourceFile, "foo(...args)");
    $tree = parse($sourceFile);

    bar();
  }

  // Foo
  function f2() {
    bar(); // does not count as comment line
  }

  // multi-line comment
  /*

  line1
  line2

  line4
  */

  function f3() {
    bar();
  }
?>

File: qlty-cli/tests/lang/python/basic.in/lines.py
def f1():
    source_file = Tempfile.new(["ruby", ".kt"])
    source_file.write("foo(*args)")
    tree = source_file.parse

    bar


# Foo
def f2():
    bar  # does not count as comment line


# TODO: We don't seem to have support for multi-line comment in python
"""

line1
line2

line4
"""


def f3():
    bar
